Siphon.ps1

```diff
@@ -0,0 +1,35 @@
+#requires -Version 1
+function Siphon-Git
+{
+    param
+    (
+        [String]
+        $Root
+    )
+    if($Root -eq $null)
```

Comment:
Better to use `if(-not($Root))` - otherwise running simply `Siphon-Git` makes it to line 16 when Test-Path throws an invalid data exception.